File: conditionals/time.py
# Josephine Chiu, Time of Day for Python
import time

current =time.time()

#get just the hour
local_time = time.localtime(current)
hour = local_time.tm_hour

if hour <=12:
    print("Good morning!")
elif hour <= 17:
    print("Good afternoon!")
else:
    print("Good evening!")

File: conditionals/time.c
// Josephine Chiu, Time of Day for C
#include <stdio.h>
#include <time.h>

int main(void){
    //current hour
    time_t now = time(NULL);
    struct tm*tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

if(hour <= 12){
    printf("Stop. It's still the morning.");
}else if(hour <= 17){
    printf("Good afternoon. The sun is still blazing.");
}else{
    printf("Good evening!");
}
    return 0;
}
